common/stash/src/main/java/com/bazaarvoice/emodb/common/stash/RestartingS3InputStream.java

```diff
@@ -38,7 +38,7 @@ public RestartingS3InputStream(AmazonS3 s3, String bucket, String key, @Nullable
 
         // Get the object synchronously so any immediate S3 errors, such as file not found, are thrown inline.
         if (range == null) {
-            s3Object = _s3.getObject(_bucket, _key);
+            s3Object = _s3.getObject(new GetObjectRequest(_bucket, _key));
```

Comment:
This doesn't change anything but it makes unit testing easier since all access to the object is through the same `getObject()` method.